pysat/utils/registry.py

```diff
@@ -50,10 +52,15 @@ def load_saved_modules():
 
 def store():
     """Rewrite user_modules.txt based on current listing"""
-    with open(os.path.join(pysat.pysat_dir, 'user_modules.txt'), 'w') as f:
+    user_modules_file = os.path.join(pysat.pysat_dir, 'user_modules.txt')
+    with Lock(user_modules_file, 'w', pysat.file_timeout) as f:
```

Comment:
This could use pysat.utils.NetworkLock

Reply:
Should we be using pysat.utils.NetworkLock by default elsewhere?